rustfmt.toml

```diff
@@ -1,2 +1,3 @@
+style_edition = "2021"
```

Comment:
Already included in #3321.

Reply:
Will automatically go away when rebasing.